src/modules/roc_netio/target_uv/roc_netio/ielement.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2019 Roc authors
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+//! @file roc_netio/target_uv/roc_netio/ielement.h
+//! @brief close handler.
+
+#ifndef ROC_NETIO_IELEMENT_H_
+#define ROC_NETIO_IELEMENT_H_
+
+#include "roc_core/list_node.h"
+#include "roc_core/refcnt.h"
+
+#include "roc_packet/address.h"
+
+namespace roc {
+namespace netio {
+
+//! Network element interface.
+class IElement : public core::RefCnt<IElement>, public core::ListNode {
+public:
+    virtual ~IElement();
+
+    //! Asynchronous stop.
+    //! @remarks
+    //!  Should be called from the event loop thread.
+    virtual void stop() = 0;
```

Comment:
If the stop is asynchronous, it would better to name it "async_stop()" or "schedule_stop()" or something. Or even better "async_close()" or "schedule_close()".

Reply:
And, accordingly, rename "stop_all_elements_()" to something like "async_close_all_elements_()".

Reply:
And thereby also rename UDPReceiver::start() UDPSender::start() to open().

Reply:
Done